vendor/github.com/Azure/azure-sdk-for-go/services/cosmos-db/mgmt/2020-04-01/documentdb/models.go

```diff
@@ -1312,6 +1312,8 @@ type DatabaseAccountCreateUpdateProperties struct {
 	IPRules *[]IPAddressOrRange `json:"ipRules,omitempty"`
 	// IsVirtualNetworkFilterEnabled - Flag to indicate whether to enable/disable Virtual Network ACL rules.
 	IsVirtualNetworkFilterEnabled *bool `json:"isVirtualNetworkFilterEnabled,omitempty"`
+	// EnableFreeTier - Enables Free Tier pricing option for the Cosmos DB account.
+	EnableFreeTier *bool `json:"enableFreeTier,omitempty"`
```

Comment:
It looks like these might have been manually added. They already exist in the structs and are causing issues with building the provider. We can run `go mod vendor` and it should fix it up